<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QWidget" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#home
{
background-color:transparent;
}

.QLineEdit

{
background : transparent; 
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;

}

.QLabel
{
	color:white;
	font-size:16px;
	qproperty-alignment: AlignCenter;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="terrain">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/stadium.png</normaloff>:/resources/Resources/stadium.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="employes">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>120</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/worker.png</normaloff>:/resources/Resources/worker.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="report">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:30px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/email.png</normaloff>:/resources/Resources/email.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="home_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="Terrain">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Terrain</string>
     </property>
    </widget>
    <widget class="QLabel" name="Mail">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="Employes">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>220</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Employes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>-20</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion terrain</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>420</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_stade">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_stade">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/delete.png</normaloff>:/resources/Resources/delete.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="adress_stade">
     <property name="geometry">
      <rect>
       <x>292</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacit√©/chaises</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>200</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>nombres d'employes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table_stade">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/load.png</normaloff>:/resources/Resources/load.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="nb_emp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="capacite_stade">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom stade</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf_2">
     <property name="geometry">
      <rect>
       <x>829</x>
       <y>410</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Export to PDF</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/pdf.png</normaloff>:/resources/Resources/pdf.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="add_stade">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>410</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>add</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/add.png</normaloff>:/resources/Resources/add.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="size">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion emplyes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>400</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_emp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="prenom_emp">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>130</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="age_emp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="salaire_emp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salaire</string>
     </property>
    </widget>
    <widget class="QComboBox" name="post_emp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maintenance</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Securit√©</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Service medicale</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>300</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>add</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/add.png</normaloff>:/resources/Resources/add.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/delete.png</normaloff>:/resources/Resources/delete.png</iconset>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="load_table">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>420</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>load data</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/load.png</normaloff>:/resources/Resources/load.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>839</x>
       <y>420</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to PDF</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/pdf.png</normaloff>:/resources/Resources/pdf.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="lineEdit_sender">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pass">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_smtp">
     <property name="geometry">
      <rect>
       <x>362</x>
       <y>170</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_465">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>210</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recepiant">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>260</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_subject">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>320</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_text">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>370</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBtn">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>460</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitBtn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>460</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>sender</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>pass</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>smtp.gmail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>465</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>to</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>sub</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="home_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
background-color:#262525;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/Resources/back.png</normaloff>:/resources/Resources/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="Background_Pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>514</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>Background_Pic</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
